app/models/concerns/parameterizable.rb

```diff
@@ -13,7 +13,15 @@ def to_param
       end
 
       def self.from_param(id)
-        self.find(id.to_i)
+        self.where(id: id.to_i).first
```

Comment:
I like this syntax, but let's be consistent with the rest of the code base & hash rockets